docker/terrarium/start_terrarium.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
```

Comment:
Check for CLOUD_PROJECT_ID existing, otherwise error on exit, see android.bash for example.